docs/cloud/cloud-troubleshooting/issue-serverless-limitations.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Serverless limitations
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 4
+---
+{% include /serverless/serverless-preview-warning.md %}
+
+# Issue - Known limitations of serverless
+
+
+-   Some SQL functionality does not work:  
+    -   `GROUP BY` on [IDSET](/docs/sql-guide/data-types/data-type-idset) and [STRINGSET](/docs/sql-guide/data-types/data-type-stringset) data types  
+    -   `PERCENTILE`  
+    -   `HAVING`  
+    -   [IDSETQ](/docs/sql-guide/data-types/data-type-idsetq) and [STRINGSETQ](/docs/sql-guide/data-types/data-type-stringsetq) data types cannot be ingested
+    -   `TOP`
+    -   `LIMIT`   
+-   There is no backup nor restore (there is a snapshot store and write-logger that holds persistent data)
+{% include /serverless/serverless-worker-shape.md %}
+{% include /serverless/serverless-worker-limits.md %}
+
+## Cause
+
+FeatureBase Serverless is in preview.
+
+## Solution
```

Comment:
Delete this and the paragraph below because the cause is enough for users to work with.

I know I say to add a solution or workaround but this is one of those edge cases where we're talking about alpha/beta functionality where stuff won't always work.